### ISSUE TITLE
Instant CTD in combat.

### ISSUE THREAD

**Issue by SimplyGhastly** [expired6978/SKSE64Plugins#7]
I converted alpha 5 to work with SSE a couple of days ago and it worked fine. I had to reinstall Skyrim on an unrelated problem and decided to use alpha 6 instead. I converted it and it opens like its supposed to, I can use it normally, presets work, everything does. That said, as soon as an enemy spots me, not even enough time for their health to appear, the game instantly ctd's, no error message. I went through my mods one by one to figure out which one it was before I found out it was racemenu. I removed and converted the file another 2 times and the problem still occurs. I went back to the alpha 5 version and it works fine.

**Comment by aers** [comment]
The sticky here:

https://www.nexusmods.com/skyrim/mods/58728?tab=posts

Applies to the skse64 version of hudextension too.

tl;dr if you want to use hudextension, move the files from interface/exported to interface
